next-sitemap.config.js

```diff
@@ -0,0 +1,15 @@
+/** @type {import('next-sitemap').IConfig} */
+module.exports = {
+  siteUrl: "https://notionpresso.com",
+  generateRobotsTxt: true,
+  // additionalPaths: [
+  //   {
+  //     href: 'https://notionpresso.com/ko',
+  //     hrefLang: 'ko',
+  //   },
+  //   {
+  //     href: 'https://notionpresso.com/en',
+  //     hrefLang: 'en',
+  //   },
+  // ]
+};
```

Comment:
p1: I forgot to remove the unnecessary annotation. I'll erase it next time.

Reply:
good